src/js/index.js

```diff
@@ -51,6 +53,7 @@ scene.add( axesHelper );
 let geometry = new THREE.SphereGeometry( 3, 32, 32 );
 let material = new THREE.MeshPhongMaterial( {color: 0xffff00, wireframe: true} );
 let sphere = new THREE.Mesh( geometry, material );
+sphere.scale.set(50,50,50);
```

Comment:
Would you be able to add the size (50) as a constant at the top of the index.js?